src/version.h

```diff
@@ -21,7 +21,7 @@ static const int GETHEADERS_VERSION = 70077;
 
 //! disconnect from peers older than this proto version
 static const int MIN_PEER_PROTO_VERSION_BEFORE_ENFORCEMENT = 70912;
```

Comment:
this needs changing to `70914` so as to not allow previously rejected protocol version again.

Reply:
k 👍